### ISSUE TITLE
networking error

### ISSUE THREAD

**Issue by pavanyejare** [terraform-google-modules/terraform-example-foundation#908]
### TL;DR

Step #1 - "tf plan validate all": ╷
Step #1 - "tf plan validate all": │ Error: Missing required argument
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │   with module.base_env.module.base_shared_vpc.module.peering_zone.google_dns_managed_zone.peering[0],
Step #1 - "tf plan validate all": │   on .terraform/modules/base_env.base_shared_vpc.peering_zone/main.tf line 42, in resource "google_dns_managed_zone" "peering":
Step #1 - "tf plan validate all": │   42:       network_url = var.target_network
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │ The argument "peering_config.0.target_network.0.network_url" is required,
Step #1 - "tf plan validate all": │ but no definition was found.
Step #1 - "tf plan validate all": ╵
Step #1 - "tf plan validate all": ╷
Step #1 - "tf plan validate all": │ Error: Missing required argument
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │   with module.base_env.module.restricted_shared_vpc.module.peering_zone.google_dns_managed_zone.peering[0],
Step #1 - "tf plan validate all": │   on .terraform/modules/base_env.restricted_shared_vpc.peering_zone/main.tf line 42, in resource "google_dns_managed_zone" "peering":
Step #1 - "tf plan validate all": │   42:       network_url = var.target_network
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │ The argument "peering_config.0.target_network.0.network_url" is required,
Step #1 - "tf plan validate all": │ but no definition was found.
Step #1 - "tf plan validate all": ╵
Finished Step #1 - "tf plan validate all"
ERROR
ERROR: build step 1 "us-central1-docker.pkg.dev/prj-b-cicd-op27/tf-runners/terraform:v1" failed: step exited with non-zero status: 21

### Terraform Configuration

```hcl
Step #1 - "tf plan validate all": ╷
Step #1 - "tf plan validate all": │ Error: Missing required argument
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │   with module.base_env.module.base_shared_vpc.module.peering_zone.google_dns_managed_zone.peering[0],
Step #1 - "tf plan validate all": │   on .terraform/modules/base_env.base_shared_vpc.peering_zone/main.tf line 42, in resource "google_dns_managed_zone" "peering":
Step #1 - "tf plan validate all": │   42:       network_url = var.target_network
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │ The argument "peering_config.0.target_network.0.network_url" is required,
Step #1 - "tf plan validate all": │ but no definition was found.
Step #1 - "tf plan validate all": ╵
Step #1 - "tf plan validate all": ╷
Step #1 - "tf plan validate all": │ Error: Missing required argument
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │   with module.base_env.module.restricted_shared_vpc.module.peering_zone.google_dns_managed_zone.peering[0],
Step #1 - "tf plan validate all": │   on .terraform/modules/base_env.restricted_shared_vpc.peering_zone/main.tf line 42, in resource "google_dns_managed_zone" "peering":
Step #1 - "tf plan validate all": │   42:       network_url = var.target_network
Step #1 - "tf plan validate all": │ 
Step #1 - "tf plan validate all": │ The argument "peering_config.0.target_network.0.network_url" is required,
Step #1 - "tf plan validate all": │ but no definition was found.
Step #1 - "tf plan validate all": ╵
Finished Step #1 - "tf plan validate all"
ERROR
ERROR: build step 1 "us-central1-docker.pkg.dev/prj-b-cicd-op27/tf-runners/terraform:v1" failed: step exited with non-zero status: 21
```

### Terraform Version

```sh
admin_@cloudshell:~/demo/gcp-networks (default-project-370310)$ terraform version
Terraform v1.3.6
on linux_amd64
```

**Comment by bharathkkb** [comment]
Hi @pavanyejare, I suspect [1] is not being pulled correctly. Per the [steps](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks-dual-svpc/envs/shared) did the shared environment run successfully?

[1] https://github.com/terraform-google-modules/terraform-example-foundation/blob/b437e29544c474d10eb3ba2abc1fd0551a7e2627/3-networks-dual-svpc/modules/base_shared_vpc/dns.tf#L52
